Cargo.toml

```diff
@@ -36,3 +36,4 @@ tokio = { git = "https://github.com/second-state/wasi_tokio.git", branch = "v1.3
 
 [features]
 default = []
+search = []
```

Comment:
I think the `search` feature should enable `llama-core/search`, like

```toml
search = ["llama-core/search"]
```

Reply:
The rag-api server search feature doesnt actually depend on `llama-core/search`, since it depends on the llamaedge-query-server which is what does the actual searching using `llama-core/search`